release/rllib_tests/rllib_tests.yaml

```diff
@@ -0,0 +1,38 @@
+- name: learning_tests
+  owner:
+    mail: "sven@anyscale.com"
+    slack: "@rllib-team"
+
+  cluster:
+    app_config: app_config.yaml
+    compute_template: 8gpus_64cpus.yaml
+
+  run:
+    timeout: 7200
+    script: python learning_tests/run_learning_tests.py
+
+- name: example_scripts_on_gpu_tests
+  owner:
+    mail: "sven@anyscale.com"
+    slack: "@rllib-team"
+
+  cluster:
+    app_config: app_config.yaml
+    compute_template: 8gpus_64cpus.yaml
+
+  run:
+    timeout: 3600
+    script: unit_gpu_tests/run.sh
```

Comment:
Does this work? Or do we need to use `bash unit_gpu_tests/run.sh`? (Is +x set automatically)?

Reply:
done